classes/microphone.dre

```diff
@@ -24,10 +24,34 @@ SOFTWARE. -->
  * @extends dr.node
  * Enables audio capture from the microphone device.
  *
+ * This example uses the microphone to monitor the audio traffic, while also
+ * amplifying the audio signal
+ *
+ *     @example
+ *     <microphone id="mic" modifystream="true" visualize="false" stream="true">
+ *       <method name="processStream" args="audio">
+ *         if (!audio) return;
+ *
+ *         var inputBuffer = audio.inputBuffer;
+ *         var left = inputBuffer.getChannelData(0);
+ *         var right = inputBuffer.getChannelData(1);
+ *
+ *         var min = Math.min.apply(null, left);
+ *	   var max = Math.max.apply(null, right);
+ *	   console.log('min', min, 'max', max);
+ *       </method>
+ *
+ *     </microphone>
+ *
  */-->
 <class name="microphone" type="coffee" clickable="true" clip="true" scriptincludes="/lib/audiorecorder/js/recorderjs/recorder.js, /lib/filesaver/FileSaver.min.js">
   
   <!--/**
+    * @attribute {Boolean} [visualize=true]
```

Comment:
I think the doc comment for this is misleading, a visualization will not be automatically be displayed, but the fft data will be available for generating a display